src/content/docs/en/pages/secure-journey/intelligent-dns/edit-intelligent-dns/edit-main-settings.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: How to configure Intelligent DNS main settings
+description: Customize your intelligent DNS domains.
+meta_tags: secure, intelligent dns, configuration, settings, domains
+namespace: docs_guides_secure_idns_configure_main_settings
+permalink: /documentation/products/guides/secure/intelligent-dns-configure-main-settings/
+menu_namespace: secureMenu
+---
+
+import Button from '~/components/Button.astro'
+import Badge from '~/components/Badge.astro'
+
+In this guide, you'll learn to create an [intelligent DNS](/en/documentation/products/intelligent-dns/) zone with its main settings and edit them after you create the zone.
+
+---
+
+## Via RTM
+
+In this section, you'll create or edit the main settings of **Intelligent DNS** with a domain via [Real-Time Manager (RTM)](https://manager.azion.com).
+
+1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > select **Intelligent DNS**.
+2. To create a new zone, click the **Add Zone** button. To edit an existing zone, select the intelligent DNS you want to edit from the list.
+
+:::caution[warning]
+Once you host your domain on Azions Intelligent DNS, you must change your domain's authoritative DNS servers to use Azion's namservers:
```

Comment:
```suggestion
Once you host your domain on Azion's Intelligent DNS, you must change your domain's authoritative DNS servers to use Azion's nameservers:
```

Reply:
I had suggested Azion's Intelligent, but I saw later on that you used Azion Intelligent... so